tests/collector_config.yaml

```diff
@@ -1,269 +1,280 @@
-        property: "config|diskSpace"
-  ClusterPropertiesCollector:
-    enum_metrics:
-      - metric_suffix: "configuration_dasConfig_admissionControlEnabled"
-        expected: "true"
-        property: "configuration|dasConfig|admissionControlEnabled"
-      - metric_suffix: "configuration_dasConfig_enabled"
-        expected: "true"
-        property: "configuration|dasConfig|enabled"
-      - metric_suffix: "configuration_drsconfig_enabled"
-        expected: "true"
-        property: "configuration|drsconfig|enabled"
-      - metric_suffix: "configuration_drsconfig_defaultVmBehavior"
-        expected: "fullyAutomated"
-        property: "configuration|drsconfig|defaultVmBehavior"
-    number_metrics:
-      - metric_suffix: "configuration_dasConfig_admissionControlPolicyId"
-        property: "configuration|dasConfig|admissionControlPolicyId"
-    info_metrics:
-      - metric_suffix: "custom_attributes_info"
-        property: "summary|customTag:INFO|customTagValue"
-  DatastorePropertiesCollector:
-    enum_metrics:
-      - metric_suffix: "summary_datastore_accessible"
-        expected: "PoweredOn"
-        property: "summary|accessible"
-  VMPropertiesCollector:
-    enum_metrics:
-      - metric_suffix: "runtime_powerState"
-        expected: "Powered On"
-        property: "summary|runtime|powerState"
-      - metric_suffix: "runtime_connectionState"
-        expected: "connected"
-        property: "summary|runtime|connectionState"
-    info_metrics:
-      - metric_suffix: "virtualdisk_scsi0_0_datastore"
-        property: "virtualDisk:scsi0:0|datastore"
-      - metric_suffix: "virtualdisk_scsi0_1_datastore"
-        property: "virtualDisk:scsi0:1|datastore"
-      - metric_suffix: "guest_os_full_name"
-        property: "config|guestFullName"
-      - metric_suffix: "guest_tools_version"
-        property: "summary|guest|toolsVersion"
-    number_metrics:
-      - metric_suffix: "summary_ethernetCards"
-        property: "summary|config|numEthernetCards"
-      - metric_suffix: "config_hardware_memory_kilobytes"
-        property: "config|hardware|memoryKB"
-  VCenterPropertiesCollector:
-    info_metrics:
-      - metric_suffix: "summary_version"
-        property: "summary|version"
-      - metric_suffix: "vc_fullname"
-        property: "summary|vcfullname"
+    - 'ClusterStatsCollector'
+    - 'ClusterPropertiesCollector'
+    - 'DatastoreStatsCollector'
+    - 'DatastorePropertiesCollector'
+    - 'HostSystemStatsCollector'
+    - 'HostSystemPropertiesCollector'
+    - 'VCenterStatsCollector'
+    - 'VCenterPropertiesCollector'
+    - 'VMPropertiesCollector'
+    - 'VMStatsCPUCollector'
+    - 'VMStatsNetworkCollector'
+    - 'VMStatsVirtualDiskCollector'
+    - 'VMStatsMemoryCollector'
+    - 'VMStatsDefaultCollector'
+
+ClusterPropertiesCollector:
+  - metric_suffix: "configuration_dasConfig_admissionControlEnabled"
+    expected: "true"
+    key: "configuration|dasConfig|admissionControlEnabled"
```

Comment:
before rolling this out, we need to make sure to reflect the changes to the configmap in helm-charts

Reply:
The deployment too. I prepared that already